packages/dev/core/src/Gizmos/positionGizmo.ts

```diff
@@ -83,6 +83,14 @@ export class PositionGizmo extends Gizmo implements IPositionGizmo {
      */
     public zPlaneGizmo: IPlaneDragGizmo;
 
+    /**
+     * all gizmos
+     */
+    get allGizmos() {
+        return this._allGizmos || (this._allGizmos = [this.xGizmo, this.yGizmo, this.zGizmo, this.xPlaneGizmo, this.yPlaneGizmo, this.zPlaneGizmo]);
+    }

```

Comment:
agree, let s not lazy load this if we can. same comment is valid in all gizmos.

Reply:
and it shouldn't be a public member

Reply:
why is it public?... And what is the improvement here as opposed to the implementation before? Now it is an extra function call that returns the same array that was defined before.

Reply:
1. Do I have to change it to a private property?
2. Now calculate the properties instead, and return the latest gizmos every time, so that when the gizmo is changed externally, it will also return the latest